src/plugins/workspace/server/utils.ts

```diff
@@ -50,3 +54,63 @@ export const getPrincipalsFromRequest = (
 
   throw new Error('UNEXPECTED_AUTHORIZATION_STATUS');
 };
+
+export const updateDashboardAdminStateForRequest = (
+  request: OpenSearchDashboardsRequest,
+  groups: string[],
+  users: string[],
+  configGroups: string[],
+  configUsers: string[]
+) => {
+  if (configGroups.length === 0 && configUsers.length === 0) {
+    updateWorkspaceState(request, {
+      isDashboardAdmin: false,
+    });
+    return;
+  }
+  const groupMatchAny = groups.some((group) => configGroups.includes(group)) || false;
```

Comment:
```suggestion
  const groupMatchAny = groups.some((group) => configGroups.includes(group));
```
groups.some will return boolean value, what is the `|| false` used for?

Reply:
I will delete the redundant code.